networkit/components.pyx

```diff
@@ -423,64 +525,87 @@ cdef class DynConnectedComponents(Algorithm):
 		self._this = new _DynConnectedComponents(G._this)
 
 	def numberOfComponents(self):
-		""" Returns the number of components.
+		"""
+		numberOfComponents()
+		
+		Returns the number of components.
 
-			Returns:
-			--------
-			count
-				The number of components.
+		Returns
+		-------
+		int
+			The number of components.
 		"""
 		return (<_DynConnectedComponents*>(self._this)).numberOfComponents()
 
 	def componentOfNode(self, v):
-		""" Returns the the component in which node @a u is.
+		"""  
+		componentOfNode(v)
+		
+		Get the the component in which node `v` is situated.
+
+		Parameters
+		----------
+		v : node
+			The node whose component is asked for.
 
-			Parameters:
-			-----------
-			v : node
-				The node.
+		Returns
+		-------
+		int
+			Component in which node `v` is situated
 		"""
 		return (<_DynConnectedComponents*>(self._this)).componentOfNode(v)
 
 	def getComponentSizes(self):
-		""" Returns the map from component to size.
+		""" 
+		getComponentSizes()
+		
+		Returns the map from component to size.
 
-			Returns:
-			--------
-			map[index, count]
-			 	A map that maps each component to its size.
+		Returns
+		-------
+		dict
+			A dict that maps each component to its size.
 		"""
 		return (<_DynConnectedComponents*>(self._this)).getComponentSizes()
 
 	def getComponents(self):
-		""" Returns all the components, each stored as (unordered) set of nodes.
+		"""
+		getComponents()
+		
+		Returns all the components, each stored as (unordered) set of nodes.
 
-			Returns:
-			--------
-			vector[vector[node]]
-				A vector of vectors. Each inner vector contains all the nodes inside the component.
+		Returns
+		-------
+		list[list[int]]
+			A list of lists. Each inner list contains all the nodes inside the component.
```

Comment:
In some cases we also use `vector[vector[int]]`. I would recommend using `list` as `vector` is not a Python type.

Reply:
Hm, based on the dict-comment above - `list(list(int))` or `list[list[int]]`. The later one would be more in line with Python lists, however in that case we should use `dict{int : int}`, right?

Reply:
I think I am leaning more towards using `(` and `)` for naming types of composite data types.

Reply:
I agree with both.

Reply:
So far we had one occasion where `[]` was used (community module). I added this fix to this PR.